### PR TITLE
Remove Actus Blockly and cleanup

### PR DESCRIPTION
**Pull request by jhbertra**
This is a pre-cleanup PR for implementing the Actus forms. This removes the blockly implementation of Actus, along with some of the now obsolete blockly stuff.

Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

**Review by palas — APPROVED** [review]
LGTM 👍

**Review by yveshauser — APPROVED** [review]
:+1:

File: marlowe-playground-client/spago.dhall [hunk 1]
```diff
@@ -58,7 +58,6 @@ You can edit this file as you like.
   , "routing-duplex"
   , "servant-support"
   , "simple-json"
-  , "string-parsers"
```
Comment (hrajchert) [inline marlowe-playground-client/spago.dhall:61]: @shmish111 is going to be happy :)

**Review by hrajchert — APPROVED** [review]
LGTM 🎉

File: marlowe-playground-client/src/Component/Blockly/State.purs [hunk 2]
```diff
@@ -165,8 +172,7 @@ handleAction (BlocklyEvent (BT.Select event)) = case newElementId event of
       $ runMaybeT do
           blocklyState <- MaybeT $ use _blocklyState
           block <- MaybeT $ liftEffect $ getBlockById blocklyState.workspace blockId
-          blockType <- MaybeT $ map pure $ liftEffect $ getBlockType block
```
[the remark this answers is not in the document]
Comment (jhbertra) [reply marlowe-playground-client/src/Component/Blockly/State.purs:168]: Yeah, I sort of went by a heuristic on this one - if pure JS functions > pure PureScript functions. In this case, the body of the function is just `return block.type;` which more or less definitely does not perform IO. Though with `defineProperty`, you can never be sure... still, I'm comfortable making that assumption.

**Pull request merged**